### ISSUE TITLE
Issues with compatibility mode and web export

### ISSUE THREAD

**Issue by SylkieGamer** [Phazorknight/Cogito#330]
**Cogito and Godot Engine Version:**
I've tested this in Godot 4.2.2 and Godot 4.3
I'm not sure the version of cogito, I've tested this with the newer main release and the newer add-on fix release along with an older version I downloaded in August. 

**Description:**
Compatibility mode in 4.2 and 4.3 runs in compatibility without having to disable the environment nodes, but as soon as you try to go from the lobby to the laboratory the transition node breaks and ends the game, I don't see any errors pop up.

Also with Web exports to itch.io the newer releases won't progress past the main menu scene. you can click on the mainmenu buttons and they react but nothing happens. 

The older version does progress past the main menu, The main lobby scene and the legacy demo scene will load, but reaching the transition node in the lobby scene to the lab scene will display a loading scene and never progress past it.

**Comment by Phazorknight** [comment]
Hey, thanks for flagging this.

The issue with not being able to progress form the main menu is flagged and tracked in https://github.com/Phazorknight/Cogito/issues/329

It's not related to the compatibility mode (or web export), at least not explicitly.

> Compatibility mode in 4.2 and 4.3 runs in compatibility without having to disable the environment nodes, but as soon as you try to go from the lobby to the laboratory the transition node breaks and ends the game, I don't see any errors pop up.

Is this in the editor? Can you activate the Cogito print log and see what messages the CogitoSceneManager prints out?

**Comment by SylkieGamer** [comment]
Yeah, it's in editor when it stops the game when entering the transition node, I've opened it in 4.3 stable with the console. It just says Connector found, moving player to ConnectorFromLobby.

Then with the Print logs in the cogito tab ticked it adds.
COGITO: InventoryUI.gd: Populate_item_grid called.
COGITO: player_hud_manager: kitchenFridgeContainer2 is in external_inventory group.
